controllers/host/platformnetworks.go

```diff
@@ -184,6 +184,11 @@ func (r *HostReconciler) IsNetworkUpdateRequired(network_instance *starlingxv1.P
 
 	if system_network != nil {
 		uuid = system_network.UUID
+	} else {
```

Comment:
if system_network == nil {
    // Network is not configured in the system, assign the pool and create the network  
     new section
}

Reply:
BTW, it is confusing for me to think about what is "system_network", can you rename it as "current_network"?

Reply:
> if system_network == nil {
> // Network is not configured in the system, assign the pool and create the network
> new section
> }
Done.

> BTW, it is confusing for me to think about what is "system_network", can you rename it as "current_network"?
Done.